src/sentry/static/sentry/less/sentry.less

```diff
@@ -37,3 +37,7 @@
 @import url('./group-similar.less');
 @import url('./component-layouts');
 @import url('./queryCount.less');
+
+// Helpers
+@import url('./includes/align.less');
+@import url('./includes/responsive-utilities.less');
```

Comment:
Moving Bootstrap-specific files here once I find that we actually use them. This should help me to see what’s intentionally used and what can be scrapped.